home/service/details.py

```diff
@@ -53,6 +69,9 @@ def _get_context(self):
             ),
             "h1_value": self.database_metadata.name,
             "is_esda": self.is_esda,
+            "is_access_requirements_url": is_access_requirements_a_url(
```

Comment:
```suggestion
            "is_access_requirements_a_url": is_access_requirements_a_url(
```
potentially an improvement